ui/webui/src/components/Error.jsx

```diff
@@ -229,6 +233,7 @@ export const CriticalError = ({ exception, isConnected, reportLinkURL }) => {
           detailsContent={exceptionInfo(exception, idPrefix)}
           buttons={[quitButton(isBootIso)]}
           isConnected={isConnected}
+          isBootIso={isBootIso}
```

Comment:
I made this change a few days ago on the purpose, to pass boot_iso vsriable through context. 
Please use:
const isBootIso = useContext(SystemTypeContext) === "BOOT_ISO";

Instead of passing the property.